java/ql/test/library-tests/dataflow/taint-jackson/dataFlow.ql

```diff
@@ -1,34 +1,4 @@
 import java
-import semmle.code.java.dataflow.DataFlow
-import semmle.code.java.dataflow.TaintTracking
-import semmle.code.java.dataflow.FlowSources
-import TestUtilities.InlineExpectationsTest
+import TestUtilities.InlineFlowTest
 
-class Conf extends TaintTracking::Configuration {
-  Conf() { this = "qltest:dataflow:jackson" }
-
-  override predicate isSource(DataFlow::Node n) {
-    n.asExpr().(MethodAccess).getMethod().hasName("taint")
-    or
-    n instanceof RemoteFlowSource
-  }
-
-  override predicate isSink(DataFlow::Node n) {
-    exists(MethodAccess ma | ma.getMethod().hasName("sink") | n.asExpr() = ma.getAnArgument())
-  }
-}
-
-class HasFlowTest extends InlineExpectationsTest {
-  HasFlowTest() { this = "HasFlowTest" }
-
-  override string getARelevantTag() { result = "hasTaintFlow" }
-
-  override predicate hasActualResult(Location location, string element, string tag, string value) {
-    tag = "hasTaintFlow" and
-    exists(DataFlow::Node src, DataFlow::Node sink, Conf conf | conf.hasFlow(src, sink) |
-      sink.getLocation() = location and
-      element = sink.toString() and
-      value = ""
-    )
-  }
-}
+class HasFlowTest extends InlineFlowTest { }
```

Comment:
It should be unnecessary to add this line. Merely importing `TestUtilities.InlineFlowTest` should be enough.